docs/connected-accounts.rst

```diff
@@ -66,6 +65,37 @@ Repository status (``repo:status``)
     but there isn't a more granular permission
     that only allows setting up SSH keys for read access.
 
+.. _github-permission-troubleshooting:
+
+GitHub Permission Troubleshooting
+`````````````````````````````````
+
+A common issue with importing a GitHub project to Read the Docs is OAuth permissions.
+Many organizations require approval for each OAuth application that is used,
+or you might have disabled it in the past for your personal account.
+This can happen at the personal or organization level,
+depending on where the project you are trying to access has permissions from.
```

Comment:
It might be helpful to discuss the outcome to the user here as well. The question I always field is "why are my repositories not in my list of repositories I can import" or "why is my organization missing from the import page". I'd like to link to this section in the dashboard UI later, so hopefully can help direct users at the source.

Reply:
Aye -- I don't know the best way to structure it, so I just put it in bold for now.